### DISCUSSION TITLE
Should I continue using Ninja ?

### DISCUSSION THREAD

**Discussion by bigusef** [vitalik/django-ninja#1197] [General]
Hi All,
First of all, Thanks to everyone who contributes to this project.
From my perspective, this project combines the best of both worlds (DRF, FastAPI). I've been using it in my main project since last year and have become increasingly reliant on it in production environments.

However, as I delve deeper, I've identified areas that need to be addressed within the project. Unfortunately, there's no information on when these areas will be covered.
There's no roadmap outlining upcoming features or their release timelines. Additionally, the documentation hasn't been updated, and there's no target date for its revision. It's been over a month since we've received any updates regarding new features or bug fixes.

Therefore, my question is: is it still safe to continue using Ninja?

**Comment by asemoon** [comment]
We are also using it and I love to have the maturity of the Django world plus the modernism of open APIs. these two are my concerns
1. The issue count is just growing. The tickets are not being closed, whether or not they need to be actioned. There are issues open from 2020. The current issues open don't get a lot of replies 
2. Lack of a roadmap as you mentioned. An official roadmap by the framework shows direction and commitment to make it better having it thriving

**Comment by bufke** [comment]
I'm invested in using this project and I'm happy to help out more. An employee of mine contributed work for async pagination. Pull requests do get reviewed and merged, even if it takes a little time. Anyone looking to contribute, vitalik has [marked help wanted issues](https://github.com/vitalik/django-ninja/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).

A long term concern I have is that Django still doesn't have async ORM (it has only sync_to_async) and it doesn't look like anyone is [working on it](https://forum.djangoproject.com/t/is-dep009-async-capable-django-still-relevant/30132/2). That limits the usefulness of this project being async friendly.

**Comment by lapinvert** [comment]
+1 It would be nice to hear from @vitalik on this subject. Maybe we can open a gofundme or something to support development.

**Comment by vitalik** [comment]
Hi @bigusef @lapinvert 

Django Ninja is enough  funded  project that is used and maintained by me inside few critical projects (and I am in contact with few companies who use it heavily).

Some projects contracted for a very long term support (think 5-10 years) - so there are legal obligations to support it at least along Django LTS schedule

That said I guess some stuff maybe a bit slower in merging, but on the other hand it is more stable/secure that way as we tend to verify changes agains some running projects

As for docs and tutorials, I am planning in near future to rework it and record a few videos (official youtube channel is here https://www.youtube.com/channel/UC5dTWMOEJlTlu5Skal9vEZg )
